### PR TITLE
Update Readme and fix the kops cluster create example

### PR DESCRIPTION
**Pull request by kmala**
**What type of PR is this?**

 /kind documentation

**What this PR does / why we need it**:
Fixes the read me links and also the kops example.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: examples/kops-new-cluster/base/cluster.yaml [hunk 1]
```diff
@@ -74,7 +74,7 @@ metadata:
     kops.k8s.io/cluster: aws-external-cloud-provider-example.k8s.local
   name: master-us-west-2a
 spec:
-  image: 099720109477/ubuntu/images/hvm-ssd/ubuntu-focal-20.04-amd64-server-20200716
```
Comment (olemarkus) [inline examples/kops-new-cluster/base/cluster.yaml:77]: Wondering if this one can be skipped altogether. kOps should use the latest stable if this one is not specified.

**Review by olemarkus — CHANGES_REQUESTED** [review]

File: examples/kops-new-cluster/base/cluster.yaml (hunk 1 above)
Comment (hakman) [reply examples/kops-new-cluster/base/cluster.yaml:77]: I think it should come from channels, or we can add the ssm way:
```
image: ssm:/aws/service/canonical/ubuntu/server/20.04/stable/current/amd64/hvm/ebs-gp2/ami-id
```

Comment (kmala) [reply examples/kops-new-cluster/base/cluster.yaml:77]: Sure... let me test these options and see

Comment (kmala) [reply examples/kops-new-cluster/base/cluster.yaml:77]: Updated to use ssm parameter.

Comment (kmala) [reply examples/kops-new-cluster/base/cluster.yaml:77]: @olemarkus can you review again when you get chance. I had addressed the comments.

**Review by olemarkus — APPROVED** [review]

**Pull request merged**